pkg/base/instance/instancemanager.go

```diff
@@ -99,7 +100,7 @@ func (im *WmiInstanceManager) QueryInstanceEx(queryString string) (*wmi.WmiInsta
 		return nil, errors.Wrapf(errors.NotFound, "Query [%s] failed with no instance", queryString)
 	}
 
-	log.Printf("[WMI] QueryInstanceEx [%s]=>[%d]instances\n", queryString, len(instances))
+	klog.V(6).Infof("[WMI] QueryInstanceEx [%s]=>[%d]instances", queryString, len(instances))
```

Comment:
@andyzhangx What would be the impact on host binaries printing to console or file. Would we need to flush the logs?